sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/ViewRegistryTest.java

```diff
@@ -7,95 +7,16 @@
-
-  @Test
-  void createBatcher_cumulative() {
-    ViewRegistry viewRegistry = new ViewRegistry();
-    InstrumentDescriptor descriptor =
-        InstrumentDescriptor.create(
-            "name", "description", "unit", InstrumentType.COUNTER, InstrumentValueType.DOUBLE);
-    MeterProviderSharedState providerSharedState =
-        MeterProviderSharedState.create(TestClock.create(), Resource.getEmpty());
-    MeterSharedState meterSharedState =
-        MeterSharedState.create(InstrumentationLibraryInfo.create("test", "1.0"));
-
-    AggregationConfiguration specification =
-        AggregationConfiguration.create(
-            AggregatorFactory.count(), MetricData.AggregationTemporality.CUMULATIVE);
-    viewRegistry.registerView(
-        InstrumentSelector.builder()
-            .setInstrumentType(InstrumentType.COUNTER)
-            .setInstrumentNameRegex("name")
-            .build(),
-        specification);
-
-    InstrumentProcessor<?> expectedInstrumentProcessor =
-        InstrumentProcessor.getCumulativeAllLabels(
-            descriptor,
-            providerSharedState,
-            meterSharedState,
-            AggregatorFactory.count().create(descriptor.getValueType()));
-
-    InstrumentProcessor<?> result =
-        viewRegistry.createBatcher(providerSharedState, meterSharedState, descriptor);
-
-    assertThat(result.generatesDeltas()).isFalse();
-    assertThat(result).isEqualTo(expectedInstrumentProcessor);
-
-    assertThat(result).isNotNull();
-  }
-
-  @Test
-  void createBatcher_delta() {
-    ViewRegistry viewRegistry = new ViewRegistry();
-
-    InstrumentDescriptor descriptor =
-        InstrumentDescriptor.create(
-            "name", "description", "unit", InstrumentType.COUNTER, InstrumentValueType.DOUBLE);
-    MeterProviderSharedState providerSharedState =
-        MeterProviderSharedState.create(TestClock.create(), Resource.getEmpty());
-    MeterSharedState meterSharedState =
-        MeterSharedState.create(InstrumentationLibraryInfo.create("test", "1.0"));
-
-    AggregationConfiguration specification =
-        AggregationConfiguration.create(
-            AggregatorFactory.count(), MetricData.AggregationTemporality.DELTA);
-    viewRegistry.registerView(
-        InstrumentSelector.builder()
-            .setInstrumentType(InstrumentType.COUNTER)
-            .setInstrumentNameRegex("name")
-            .build(),
-        specification);
-
-    InstrumentProcessor<?> expectedInstrumentProcessor =
-        InstrumentProcessor.getDeltaAllLabels(
-            descriptor,
-            providerSharedState,
-            meterSharedState,
-            AggregatorFactory.count().create(descriptor.getValueType()));
-
-    InstrumentProcessor<?> result =
-        viewRegistry.createBatcher(providerSharedState, meterSharedState, descriptor);
-
-    assertThat(result.generatesDeltas()).isTrue();
-    assertThat(result).isEqualTo(expectedInstrumentProcessor);
-
-    assertThat(result).isNotNull();
-  }
-
```

Comment:
Did we lose coverage here?  Should this have moved to the InstrumentProcessorTest instead?

Reply:
Already tested when we test all the instrument types